### PR TITLE
New Post - Testing web design color contrast

### LINKED ISSUE #8062: [A11y][CSS][DevTools] Testing Web Design Contrast
* Proposed title: Testing Web Design Contrast
* Proposed by: argyle
* Related areas: Accessibility,CSS,Chrome DevTools
* Content type: How-To/API Guide (https://documentation.divio.com/how-to-guides/)
* [Draft](https://docs.google.com/document/d/1BSWE83huQ2tjoUl8Y0OlUmc3xKmBabVbVR_jmvhpbV8/edit#)
* Content summary: One of the most common accessibility issues is with improper contrasting text and/or colors. This post will focus exclusively on tools for testing contrast so that developers are prepared for easy checking and advanced checking.

### PR DESCRIPTION
**Pull request by argyleink**
Fixes #8062

To Do
- [x] alt text for all assets
- [x] final reviews
- [x] add custom hero image

### CODE REVIEW

File: src/site/_data/authorsData.json [hunk 1]
```diff
@@ -1688,5 +1688,10 @@
     "homepage": "https://davidea.st",
     "twitter": "_davideast",
     "image": "image/vS06HQ1YTsbMKSFTIPl2iogUQP73/S8CN1AfcJpoUJvYqrfeT.jpeg"
+  },
+  "superhighfives": {
+    "homepage": "https://t.co/IGOafTKR29",
+    "twitter": "superhighfives",
```
Comment (superhighfives) [inline src/site/_data/authorsData.json:1694]: It'd be great to add my GitHub here: https://github.com/superhighfives

File: src/site/_data/authorsData.json [hunk 2]
```diff
@@ -1688,5 +1688,10 @@
     "homepage": "https://davidea.st",
     "twitter": "_davideast",
     "image": "image/vS06HQ1YTsbMKSFTIPl2iogUQP73/S8CN1AfcJpoUJvYqrfeT.jpeg"
+  },
+  "superhighfives": {
+    "homepage": "https://t.co/IGOafTKR29",
```
Comment (superhighfives) [inline src/site/_data/authorsData.json:1693]: Can this just be a direct link to https://charliegleason.com/?

Comment (argyleink) [reply src/site/_data/authorsData.json:1693]: def, looks like a copy n pasta error 👍🏻

**Comment by argyleink** [comment]
this is ready for final reviews!

File: src/site/content/en/blog/testing-web-design-color-contrast/index.md [hunk 3]
```diff
@@ -0,0 +1,433 @@
...
+layout: post
+title: Testing Web Design Color Contrast
+subhead: |
+  An overview of three tools and techniques for testing and verifying accessible color contrast of your design.
+description: |
+  An overview of three tools and techniques for testing and verifying accessible color contrast of your design.
+authors:
+  - adamargyle
+  - superhighfives
+date: 2022-09-16
```
Comment (rachelandrew) [inline src/site/content/en/blog/testing-web-design-color-contrast/index.md:11]:
```suggestion
date: 2022-09-22
```

File: src/site/content/en/blog/testing-web-design-color-contrast/index.md [hunk 4]
```diff
@@ -0,0 +1,433 @@
...
+hero: image/vS06HQ1YTsbMKSFTIPl2iogUQP73/q6qpUvm2TwTyRWKbYCQd.png
+thumbnail: image/vS06HQ1YTsbMKSFTIPl2iogUQP73/93UiiPbes6SjeGjB8w1N.png
+alt: A description of the hero image for screen reader users.
```
Comment (rachelandrew) [inline src/site/content/en/blog/testing-web-design-color-contrast/index.md:14]:
```suggestion
alt: Text becoming progressively lower contrast.
```

File: src/site/content/en/blog/testing-web-design-color-contrast/index.md [hunk 5]
```diff
@@ -0,0 +1,433 @@
...
+While all of the examples may be readable to you, this isn't the case for
+everyone.
+
+Accessible color contrast is a practice that ensures text is readable for
+everyone. Sometimes testing contrast is easy and sometimes it's really hard. By
+the end of this post you'll have three new tools and techniques for inspecting,
+correcting, and verifying your web design contrast so you can tackle the hardest
+scenarios.
+
+## WCAG and Color Contrast
```
Comment (rachelandrew) [inline src/site/content/en/blog/testing-web-design-color-contrast/index.md:35]:
```suggestion
## WCAG and color contrast
```

**Review by rachelandrew — APPROVED** [review]
Approved with a couple of small nits plus updated the date to today. This can go out today (22nd) if you are happy?.

File: src/site/content/en/blog/testing-web-design-color-contrast/index.md [hunk 6]
```diff
@@ -0,0 +1,433 @@
...
+## Testing contrast between colors
+
+So, now that we know what we’re looking for, how do we test this? Here are three
+free tools to assist you in inspecting, correcting and measuring your web site's
+contrast. The strengths and weaknesses of each will be outlined so you can
+confidently test the accessibility of your site’s colors and content in a
+multitude of ways.
+
+1. [Pika](https://superhighfives.com/pika)<br>
+A MacOS system extension, uniquely capable of showing the contrast of any colors
```
Comment (superhighfives) [inline src/site/content/en/blog/testing-web-design-color-contrast/index.md:107]: I'd go with "A MacOS app, uniquely..." as it's not technically a system extension.

**Comment by argyleink** [comment]
> Approved with a couple of small nits plus updated the date to today. This can go out today (22nd) if you are happy?.

ready to go out! we're happy 🙂

**Pull request merged**